trappy/plotter/js/ILinePlot.js

```diff
@@ -123,6 +123,7 @@ var ILinePlot = ( function() {
             stepPlot: t_info.step_plot,
             logscale: t_info.logscale,
             fillGraph: t_info.fill_graph,
+            fillAlpha: t_info.fill_alpha,
```

Comment:
Hmm, it works on my FF 49.0 on Ubuntu 14.04. Perhaps try deleting `~/.local/share/jupyter/nbextensions/` and force-refreshing (`ctrl`+`shift`+`R`)?